app/views/responsible_body/devices/schools/show.html.erb

```diff
@@ -7,7 +7,13 @@
       <%= @school.name %>
     </h1>
 
-    <%- unless @school.has_std_device_allocation? %>
+    <%- if @school.has_std_device_allocation? %>
+      <%= render DeviceCountComponent.new(
+        max_count: @school.std_device_allocation.cap,
+        ordered_count: @school.std_device_allocation.devices_ordered,
+        show_action: @school.std_device_allocation.devices_ordered < @school.std_device_allocation.cap,
```

Comment:
Can we use the school helper methods here? (I think this should show the same thing?)
```suggestion
        show_action: @school.can_order_devices_right_now?,
```